pkg/runtime/config.go

```diff
@@ -50,6 +50,8 @@ const (
 	DefaultGracefulShutdownDuration = time.Second * 5
 	// DefaultAppHealthCheckPath is the default path for HTTP health checks.
 	DefaultAppHealthCheckPath = "/health"
+	// Metadata needed to call bulk subscribe
+	BulkSubscribe = "bulkSubscribe"
```

Comment:
Because this isn't a config option, can you put it in the runtime.go file?

Reply:
Moved to runtime.go